src/main/kotlin/com/avail/compiler/AvailCompiler.kt

```diff
@@ -4555,9 +4588,9 @@ class AvailCompiler(
 					return
 				}
 				val parts1 = ArrayList<A_Phrase>()
-				phrase1.value.childrenDo(Consumer { parts1.add(it) })
+				phrase1.value.childrenDo { parts1.add(it) }
 				val parts2 = ArrayList<A_Phrase>()
-				phrase2.value.childrenDo(Consumer { parts2.add(it) })
+				phrase2.value.childrenDo { parts2.add(it) }
```

Comment:
I believe this can be reduced further to ...childrenDo(parts1::add)  (and parts2)

Reply:
Fixed.